src/components/Scorecard.tsx

```diff
@@ -16,25 +15,13 @@ type Props = {
   ) => void;
 };
 
-const useStyles = makeStyles(() => {
-  return {
-    checkbox: {
-      width: "10px",
-      height: "10px",
-      transform: "translateY(-2px)",
-      marginRight: "4px",
-    },
-  };
-});
-
```

Comment:
There are a lot of weird things happening because we have two different versions of MUI running side by side, but the CSS changes seem to get us across the line even though it's not where we want to be long-term.